docs/python/reference/task_queues.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 4
+title: Task Queues
+---
+
+Task queues allows you to submit tasks to run in the background with a controlled degree of parallelism.
+They are useful for orchestration of parallel events.
+
+### class dbos.Queue
+
+```python
+Queue(
+    name: str = None,
+    concurrency: Optional[int] = None,
+)
+```
+
+**Parameters:**
+- `name`: The name of the queue. Must be unique among all your queues.
+- `concurrency`: The maximum number of tasks from this queue that may run concurrently.
+This concurrency limit is global across all DBOS processes using this queue.
+If not provided, any number of tasks may run concurrently.
+
+
+### enqueue
+
+```python
+queue.enqueue(
+    func: Workflow[P, R],
```

Comment:
This `Workflow[P, R]` may cause people to think you can only enqueue workflows. Explain it includes steps and transactions as well.